Signal/src/environment/Environment.m

```diff
@@ -244,6 +244,8 @@ - (NotificationsManager *)notificationsManager
 
 + (PropertyListPreferences *)preferences
 {
+    OWSAssert([Environment getCurrent] != nil);
+    OWSAssert([Environment getCurrent].preferences != nil);
```

Comment:
This should help us catch most mistakes of this form going forward. It won't work for release-only bugs (like this one) even if we test release builds, since asserts don't fire in production.  I'm thinking about how to improve our asserts around this.